tox.ini

```diff
@@ -4,7 +4,7 @@
 # E128: Line continuation not lined up
 # E402: Imports can be spaced out
 # W503: Making if's readable
-ignore = E121,E128,E402,W503,E12
+ignore = E121,E128,E402,W503,E12,C901
```

Comment:
Not sure why this cause `flake8` to complain but this seems fine for the time being

Reply:
so i dont forget, i created an issue to resolve this: https://github.com/drewmullen/vault-kv-migrate/issues/32

its not a priority but, at least the use is documented